doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
```

Comment:
I spent days trying to find an Official Docker Image that could be used as the front-end to another one.  I really want to show Istio being used to connect two existing microservices.  Providing our own front-end image, even 20 lines of Python, seemed to add complexity.  Can you think of an image that would not confuse the end-user, and would also let the end user not have to think about the contents of the image?